modules/opentelemetry/src/main/scala/zio/opentelemetry/OpenTelemetry.scala

```diff
@@ -0,0 +1,81 @@
+package zio.opentelemetry
+
+import java.util.concurrent.TimeUnit
+
+import io.opentelemetry.trace._
+import zio.clock.Clock
+import zio.{ FiberRef, UIO, URIO, URLayer, ZIO, ZLayer }
+import zio.opentelemetry.SpanUtils.endSpan
+
+object OpenTelemetry {
```

Comment:
Given that OpenTelemetry also supports metrics: https://github.com/open-telemetry/opentelemetry-java/blob/master/QUICKSTART.md#metrics, this might better be called `Tracer`?

In the future, we could then have 2 modules, each named after the provider on the java OpenTelemetry:
`getMeterProvider` -> `zio.opentelemetry.Meter`
`getTracerProvider` -> `zio.opentelemetry.Tracer`